app/build.gradle

```
@@ -164,7 +164,7 @@ dependencies {
         exclude module: 'support-annotations'
     }
 
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:5ac80624a40f4c600ae493e66881b5bf008f0ddb'
+    implementation 'com.github.TeanNewPipe:NewPipeExtractor:6633f26ec5a73a8e932de575b7a0643b6ad6c890'
```

```suggestion
    implementation 'com.github.TeamNewPipe:NewPipeExtractor:6633f26ec5a73a8e932de575b7a0643b6ad6c890'
```

Please fix this. Otherwise your build won't succeed. Please squash the commits afterwards.

---

I already noticed and fixed this. I really need more sleep.